vita3k/app/src/app_init.cpp

```diff
@@ -182,6 +198,7 @@ bool init(EmuEnvState &state, Config &cfg, const Root &root_paths) {
     state.config_path = string_utils::utf_to_wide(root_paths.get_config_path_string());
     state.cache_path = string_utils::utf_to_wide(root_paths.get_cache_path_string());
     state.shared_path = root_paths.get_shared_path_string();
+    state.static_assets_path = root_paths.get_shared_path();
```

Comment:
Is this correct? I thought it should be get_static_assets_path() here?

Reply:
indeed, my bad